tests/test_calc_cellvol.cpp

```diff
@@ -0,0 +1,84 @@
+//===========================================================================
```

Comment:
The entire top section here is copied from another file and should be cleaned up.
 - In general, we no longer include the part with Authors etc, relying on git to do that better.
 - The copyright part should refer to OPM, not OpenRS (which never happened...).
 - I guess the copyright here should be 2018 Statoil (or some horse-related company name...) only.